src/js/sa11y.js

```diff
@@ -1294,40 +1303,41 @@ class Sa11y {
                 <span class='sa11y-outline-list-item sa11y-yellow-text sa11y-bold'>${htext}</span>
             </li>`;
 
-                if ($el.not(this.options.outlineIgnore).length !== 0) {
+            let ignoreArray = [];
+            if (outlineIgnore) {
+                ignoreArray = Array.from(document.querySelectorAll(outlineIgnore));
+            }
 
+            if (!ignoreArray.includes(el)) {
                     //Append heading labels.
-                    $el.not(this.options.outlineIgnore).append(
-                        `<span class='sa11y-heading-label'>H${level}</span>`
-                    );
+                    el.insertAdjacentHTML("beforeend", `<span class='sa11y-heading-label'>H${level}</span>`);
 
                     //Heading errors
-                    if (error != null && $el.closest("a").length > 0) {
+                    if (error != null && el.closest("a").length != null) {
```

Comment:
This cause a second error. In vanila js `.closest(blabla)` will return `null` if nothing found. Current code made in jquery style.

Change the IF condition to:
```
if (error != null && el.closest("a")) {
```